RPIPs/RPIP-43.md

```diff
@@ -99,18 +100,15 @@ into shares is defined in [RPIP-46](RPIP-46.md).
     surplus disposition contract
   - If `debt` exists when called, the remaining rewards SHALL first be used to pay off `debt`
   - When called, any remaining rewards SHALL then be held in the megapool as unclaimed node operator funds 
-  - This function SHALL allow any user to call it
+  - This function SHALL be permissionless
   - If called by the node operator, this function SHOULD claim all unclaimed node operator funds
 - There SHALL be a capital distribution function in the megapool
   - When called, capital borrowed from the protocol that has been released from
     exited validators SHALL be sent to the rETH contract.
     - If the capital is insufficient to repay the protocol, the shortfall SHALL be added to `debt`
   - When called while the megapool has `debt`, the remaining capital from exited validators SHALL first be used to pay off `debt`
   - When called, the remaining capital SHALL then be held in the megapool as unclaimed node operator funds
-  - This function SHALL allow any user to call it following a mandatory time
-    delay configurable by the pDAO. The delay SHALL be initialized by a
-    `startUserDistribute` function. After the delay is complete, any user may
-    then call the capital distribution fuction.
+  - This function SHALL be permissionless following a mandatory time delay configurable by the pDAO. The delay SHALL be initialized by a `startUserDistribute` function. After the delay is complete, any user may then call the capital distribution function.
```

Comment:
You're right - should be windowed

Reply:
>   - This function SHALL support permissionless use, with additional restrictions:
    - There SHALL be pDAO settings called `userDistributeDelay` (initially set to 30 days) and `userDistributeWindow` (initially set to 60 days)
    - A permissionless `startUserDistribute` function SHALL record the time when called, as long as there is no previously recorded time _or_ the previously recorded time was longer than `userDistributeWindow` ago
    - The capital distribution function may be called permissionlessly if `userDistributeDelay` has passed since the `startUserDistribute` call, but `userDistributeWindow` has not yet passed since the `startUserDistribute` call

How's this?